files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1150,7 +1150,7 @@ MyApplet.prototype = {
         this.settings.bind("activate-on-hover", "activateOnHover", this._updateActivateOnHover);
         this._updateActivateOnHover();
 
-        this.menu.actor.add_style_class_name('menu-background');
+        this.menu.setCustomStyleClass('menu-background');
```

Comment:
Don't applets already have the ability to do this?

The weather applet does (because I requested it) and some of your applets @collinss do too (also because I opened a request.)

In many applets one can utilize this for some really cool effects.